projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/test/component.js

```diff
@@ -0,0 +1,171 @@
+			.stub( recommendationsActions, 'updateRecommendationsStep' )
+			.returns( DUMMY_ACTION );
+	} );
+
+	after( function () {
+		updateRecommendationsStepStub.restore();
+	} );
+
+	describe( 'Monitor', () => {
+		const stepSlug = 'monitor';
+
+		it( 'shows the title of the Feature Prompt component', () => {
+			render( <FeaturePrompt stepSlug={ stepSlug } />, {
+				initialState: buildInitialState( { recommendationsStep: stepSlug } ),
+			} );
+			expect(
+				screen.getByRole( 'heading', {
+					name: /Would you like Downtime Monitoring to notify you if your site goes offline?/i,
+				} )
+			).to.be.not.null;
+		} );
+
+		it( 'shows the enable and skip feature buttons', () => {
+			render( <FeaturePrompt stepSlug={ stepSlug } />, {
+				initialState: buildInitialState( { recommendationsStep: stepSlug } ),
+			} );
+
+			expect(
+				screen.getByRole( 'link', {
+					name: /Enable Downtime Monitoring/i,
+				} )
+			).to.be.not.null;
+
+			expect(
+				screen.getByRole( 'link', {
+					name: /Not now/i,
+				} )
+			).to.be.not.null;
+		} );
+
+		it( 'calls the right actions when a user clicks on the enable feature button', () => {
+			// Stub methods that perform side-effects through async actions
+			const recordEventStub = sinon.stub( analytics.tracks, 'recordEvent' );
+			const addSelectedRecommendationStub = sinon
+				.stub( recommendationsActions, 'addSelectedRecommendation' )
+				.returns( DUMMY_ACTION );
+			const activateFeatureStub = sinon.stub().returns( DUMMY_ACTION );
+			const mapDispatchToPropsStub = sinon.stub( featureUtils, 'mapDispatchToProps' ).returns( {
+				activateFeature: activateFeatureStub,
+			} );
+
+			render( <FeaturePrompt stepSlug={ stepSlug } />, {
+				initialState: buildInitialState( { recommendationsStep: stepSlug } ),
+			} );
+
+			const enableFeatureButton = screen.getByRole( 'link', {
+				name: /Enable Downtime Monitoring/i,
+			} );
+			expect( enableFeatureButton ).to.be.not.null;
+
+			// Make sure the enable button points to the right link
+			expect( enableFeatureButton.href ).to.have.string( 'recommendations/related-posts' );
+
+			expect( recordEventStub.callCount ).to.be.equal( 0 );
+			fireEvent.click( enableFeatureButton );
+
+			// Make sure tracks work
+			expect(
+				recordEventStub.withArgs( 'jetpack_recommended_feature_enable_click', {
+					feature: stepSlug,
+				} ).callCount
+			).to.be.equal( 1 );
+
+			// Make sure addSelectedRecommendation action is called with the right step slug
+			expect( addSelectedRecommendationStub.withArgs( stepSlug ).callCount ).to.be.equal( 1 );
+
+			// Make sure activateFeature action is called
+			expect( activateFeatureStub.callCount ).to.be.equal( 1 );
+
+			// Restore stubs
+			recordEventStub.restore();
+			addSelectedRecommendationStub.restore();
+			mapDispatchToPropsStub.restore();
+		} );
+
+		it( 'calls the right actions when a user clicks on the skip feature button', () => {
+			const DUMMY_ACTION = { type: 'dummy' };
```

Comment:
This is already declared on line 42

```suggestion

```

Reply:
Good catch! I'm definitely removing that unneccesary `const` definition.